API.md

```diff
@@ -1563,11 +1563,29 @@ If you need to communicate between callback functions for any purpose, you can u
 
 the values `string_body` and `json_body` have been removed from the structures `struct _u_request` and `struct _u_response`. This may be painless in the response if you used only the functions `ulfius_set_xxx_body_response`. Otherwise, you should make small arrangements to your code.
 
+### GNU TLS extension
+Ulfius GNU TLS extension allows sending HTTPS requests with client certificate and enables additional client certificate validation option for HTTS server.
+
+Using websocket requires [libgnutls](https://www.gnutls.org/).
+
+This extension can be disabled with `GNUTLSFLAG` option when using make:
+
+```shell
+$ make GNUTLS=1 WEBSOCKETFLAG=1
```

Comment:
I'd use the `INSTALL.md` to explain Makefile options, I prefer `API.md` to be read after installation.